modules/ROOT/pages/docker/docker-compose-standalone.adoc

```diff
@@ -0,0 +1,108 @@
+:description: Running Neo4j in a Docker container using Docker Compose
+:page-role: enterprise-edition new-4.4.38
+
+[[docker-compose-neo4j-standalone]]
+= Deploy a Neo4j standalone server using Docker Compose
+
+You can deploy a Neo4j standalone server using Docker Compose by defining the container configuration in a _docker-compose.yml_ file and authenticating with basic authentication or Docker secrets.
+
+[[docker-compose-basic-authentication]]
+== Deploy a Neo4j server using basic authentication mechanism
+
+Before you start, verify that you have installed Docker Compose.
+For more information, see the https://docs.docker.com/compose/install/[Install Docker Compose official documentation].
+
+. Create a project folder where you will store your _docker-compose.yml_ file and run your Neo4j server.
+. Prepare your _docker-compose.yml_ file using the following example.
+For more information, see the Docker Compose official documentation on https://docs.docker.com/compose/compose-file/#service-configuration-reference[Docker Compose specification].
++
+.Example of a _docker-compose.yml_ file
+[source,yaml,subs="attributes+,+macros"]
+----
+services:
+  neo4j:
+    image: neo4j:latest
```

Comment:
This version should probably be `{neo4j-version-exact}` like in the other sections? Or the latest 4.4

Reply:
Yes, it should be `{neo4j-version-exact}`.